tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+                    "type": "WindowConstructionAbridged",
+                    "name": "Generic Single Pane Window",
+                    "layers": [
+                        "Generic Clear Glass"
+                    ]
+                },
+                {
+                    "type": "OpaqueConstructionAbridged",
+                    "name": "Generic Exposed Floor Slab",
+                    "layers": [
+                        "Generic Metal Surface",
+                        "Generic Ceiling Air Gap",
+                        "Generic 50mm Insulation Board",
+                        "Generic Lightweight Concrete"
+                    ]
+                },
+                {
+                    "type": "OpaqueConstructionAbridged",
+                    "name": "Generic Underground Wall",
+                    "layers": [
+                        "Generic 50mm Insulation Board",
+                        "Generic Heavyweight Concrete"
+                    ]
+                },
+                {
+                    "type": "OpaqueConstructionAbridged",
+                    "name": "Generic Interior Floor",
+                    "layers": [
+                        "Generic Acoustic Tile",
+                        "Generic Ceiling Air Gap",
+                        "Generic Lightweight Concrete"
+                    ]
+                },
+                {
+                    "type": "WindowConstructionAbridged",
+                    "name": "Generic Double Pane Window",
+                    "layers": [
+                        "Generic Clear Glass",
+                        "Generic Window Air Gap",
+                        "Generic Clear Glass"
+                    ]
+                },
+                {
+                    "type": "OpaqueConstructionAbridged",
+                    "name": "Generic Interior Wall",
+                    "layers": [
+                        "Generic Gypsum Board",
+                        "Generic Wall Air Gap",
+                        "Generic Gypsum Board"
+                    ]
+                },
+                {
+                    "type": "OpaqueConstructionAbridged",
+                    "name": "Generic Exterior Door",
+                    "layers": [
+                        "Generic Metal Surface",
+                        "Generic 25mm Insulation Board",
+                        "Generic Metal Surface"
+                    ]
+                },
+                {
+                    "type": "WindowConstructionAbridged",
+                    "name": "Triple Pane Window",
+                    "layers": [
+                        "Generic Clear Glass",
+                        "Generic Window Air Gap",
+                        "Generic Clear Glass",
+                        "Generic Window Air Gap",
+                        "Generic Clear Glass"
+                    ]
+                }
+            ],
+            "materials": [
+                {
+                    "type": "EnergyMaterial",
```

Comment:
There is no abridged version of an EnergyMaterial?  Abridged means "references external objects" and non-abdriged means "all properties nested"?

Reply:
Yes, "Abridged" means that not all of the information needed to completely re-serialize the object to its Python version is contained within the dictionary.  In other words, there are references to other objects by name and not as nested children objects.  Because Materials always have all of the information to re-create them, there's no need for an Abridged schema for them.

Reply:
I am going to resolve this now that it has been answered.